salt/modules/cmdmod.py

```diff
@@ -3231,12 +3231,16 @@ def run_chroot(root,
     '''
     __salt__['mount.mount'](
         os.path.join(root, 'dev'),
-        'udev',
```

Comment:
Is this change going to create a compatibility issue with some systems?

Reply:
AFAIK this can be any random text as there is not real device named 'udev', nor 'devtmpfs'. Is only the type what is important.

Distributions normalize it to the same name of the type, so a `mount` will show lines like

```
devtmpfs on /dev type devtmpfs ...
tmpfs on /run type tmpfs ...
...
```

So always following `"type" on "mount point" type "type"`, duplicating the name without problems.

Reply:
Thank you for clarifying so I did not have to dig into it more :)